### ISSUE TITLE
New test failure on Cygwin

### ISSUE THREAD

**Issue by arstrube** [cpputest/cpputest#798]
```c++
./CppUTestTests
..................................................
..................................................

../tests/MemoryLeakDetectorTest.cpp:247: error: Failure in TEST(MemoryLeakDetectorTest, TwoAllocOneF
reeOneLeak)
        CHECK(!output.contains("size: 4")) failed

..................................................
...........................................!......
..................................................
..................................................
..................................................
..................................................
..................................!....!.!......!.
.........!..!..!..!..!..!..!..!..!..!..!..!..!..!.
.!..!..!..!..!..!..!..!......!..!..!..!..!..!..!..
!.............!...............!.........
Errors (1 failures, 590 tests, 553 ran, 1009 checks, 37 ignored, 0 filtered out, 281 ms)
````

**Comment by arstrube** [comment]
This is one of the funniest bugs I've seen lately. It fails, because `size: 4` is part of the memory dump of the memory leak, which obviously re-used the space containing this string:
```
../tests/MemoryLeakDetectorTest.cpp:245: error: Failure in TEST(MemoryLeakDetectorTest, TwoAllocOneF
reeOneLeak)
        Memory leak(s) found.
Alloc num (2) Leak size: 12 Allocated at: <unknown> and line: 0. Type: "alloc"
        Memory: <0x8004c950> Content:
    0000: 73 69 7a 65 3a 20 34 00  00 75 6e 6b             |size: 4..unk|
Total number of leaks:  1
```
I am going to submit a pull request which checks for `Leak size: 4` rather than just `size: 4`

**Comment by arstrube** [comment]
@basvodde a very interesting bug and a very simple fix - you did see this, right?

**Comment by arstrube** [comment]
This issue has been fixed by PR #800.
